src/lib.rs

```diff
@@ -51,6 +52,7 @@ mod texture;
 #[cfg(feature = "opengl")]
 pub mod window;
 
+pub use audio::{Clip, Source};
```

Comment:
It might not be a good idea to export these since it's not clear what `Clip` and `Source` mean on their own. In the upcoming code for animations, there is also a `Clip` type which will clash.

Reply:
Reasonable